src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -20,6 +20,61 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
 
             if (context.Operation == ExtensionContext.OperationKind.LocalRun)
             {
+                // default placement port number
+                var placementPort = 50005;
+                var isCustomPlacementPort = false;
+
+                var existingDaprPlacementDefinition = context.Application.Services.FirstOrDefault(s =>
+                    s is ContainerServiceBuilder serviceBuilder && serviceBuilder.Image == "daprio/dapr");
```

Comment:
In case of using ImageName + Tag, placement definition cannot be found.
![image](https://user-images.githubusercontent.com/1976118/99438900-c409e680-2914-11eb-9290-925c9e71f7fc.png)

Reply:
The more I think about this, the more I don't like sniffing to determine if the placement port is already defined or not.

I think including an option called `excludePlacementService`, which default to false, is enough to have people either include or exclude it as required.

Reply:
I have removed sniffing. It will now use an auto assigned port and be agnostic about the user defined services. The only way to control the port is via the `placement-port` property.